src/event.rs

```diff
@@ -1301,3 +1062,36 @@ pub enum MouseScrollDelta {
     /// and move the content right and down (to reveal more things left and up).
     PixelDelta(PhysicalPosition<f64>),
 }
+
+/// Handle to synchroniously change the size of the window from the
+/// [`WindowEvent`].
+#[derive(Debug, Clone)]
+pub struct InnerSizeWriter {
```

Comment:
Perhaps this makes sense to put in the `dpi` module?

Reply:
It's a part of the event, I'm not sure it makes sense in other module.